app/controllers/admin/official_postcodes_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Admin
+  class OfficialPostcodesController < Admin::ApplicationController
```

Comment:
@leoapost I thought about this before I named this model and the same name occur to me too but we are using 'postcode' in all over the place and it's not relevant to OfficialPostode but just to postcode inserted by a user. OfficialPostode means that data in the model are from the official source and can be relied on. I'm happy to discuss this further if you wish :)